schemas/os.yml

```
@@ -13,6 +13,20 @@
   type: group
   fields:
 
+    - name: type
+      level: extended
+      type: keyword
+      short: 'Which commercial OS family (one of: linux, macos, unix or windows).'
+      description: >
+        Use the `os.type` field to categorize the operating system in one of
+        the broad commercial families.
+
+        One of these following values should be used (lowercase): linux, macos, unix, windows.
+
+        If the OS is not part of any of this list, the field should not be populated.
+        Please let us know by opening an issue with ECS, to have it added to the list.
```

```suggestion
        Please let us know by opening an issue with ECS, to propose adding it to the list.
```
Minor adjustment.

I think this wording better primes users to understand any additional values will need to be discussed and mutually agreed on.

---

Good idea. It's always good to know in advance one's dreams *might* be shattered 😂